### ISSUE TITLE
Remove clj-time (and Joda-Time) dependency

### ISSUE THREAD

**Issue by Frozenlock** [dscarpetti/codax#31]
`Joda-Time` was nice a few years back, but nowadays Clojure users are expected to use `java.util.Date` (mostly via `#inst`) and `java.time.instant`.

Most importantly, `clj-time` is deprecated. It's a little annoying to bring those dependencies in new projects.

**Comment by dscarpetti** [comment]
Yeah, I agree. It's pretty simple to add support manually for a legacy databases that require it. I'd like to include this in the 1.4.0 release. I've made the changes in PR #33, I'd appreciate it if you could take a quick look.

**Comment by Frozenlock** [comment]
Looks good to me :+1:

**Comment by dscarpetti** [comment]
merged and deployed as version 1.4.1

**Comment by dscarpetti** [comment]
🤦 fixed and deployed as 1.4.2. thanks

**Comment by dscarpetti** [comment]
...and now 1.4.3 fixes the tests by removing references to clj-time
